src/shared/compatibility.js

```diff
@@ -223,6 +241,24 @@ const hasDOM = typeof window === 'object' && typeof document === 'object';
 
 } // End of !PDFJSDev.test('CHROME')
 
+// Provides support for String.prototype.padStart in legacy browsers.
+// Support: IE, Chrome<57
+(function checkStringPadStart() {
+  if (String.prototype.padStart) {
+    return;
+  }
+  require('core-js/fn/string/pad-start');
+})();
+
+// Provides support for String.prototype.padStart in legacy browsers.
```

Comment:
Nit: padEnd. Also, we don't use padding in this patch, so why add these polyfills? I understand that since we use `startsWith`, `endsWith` is useful to have as well, but we don't use any of the padding functions as far as I can tell.

Reply:
Too much copy-paste, thanks!

> but we don't use any of the padding functions as far as I can tell.

Actually we do, please see https://github.com/mozilla/pdf.js/blob/af31b980b0db7b110965cdb468ee608769835d1e/src/display/dom_utils.js#L403